frontend/packages/lm-eval/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "private": true,
+  "name": "@odh-dashboard/lm-eval",
+  "description": "Language Model Evaluation plugin.",
+  "version": "0.0.0",
+  "scripts": {
+    "update-subtree": "../../scripts/package-subtree.sh lm-eval"
+  },
+  "subtree": {
+    "repo": "https://github.com/trustyai-explainability/trustyai-dashboard.git",
+    "src": "",
+    "target": "upstream",
+    "commit": "679fadce6fd49e2a9574c586bbaa49d4cf5f7542"
+  },
+  "module-federation": {
+    "name": "lmEval",
+    "remoteEntry": "/remoteEntry.js",
+    "authorize": true,
+    "tls": false,
+    "proxy": [
+      {
+        "path": "/lm-eval-api",
+        "pathRewrite": "/api"
+      }
+    ],
+    "local": {
+      "host": "localhost",
+      "port": 9001
+    },
+    "service": {
+      "name": "lm-eval-ui-service",
```

Comment:
You can keep this, probably llmeval is gonna be the first one with a proper service, just commenting to do a full review.